Job periodic-hco-push-nightly-build-master fails to build pyyaml dependency

**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
>
/kind bug
> /kind enhancement


**What happened**:
[Prow job](https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/logs/periodic-hco-push-nightly-build-master/1328881349393649664)

```
Requirement already satisfied: setuptools in /usr/local/lib/python2.7/dist-packages (from yamllint) (41.0.1)
Building wheels for collected packages: pyyaml
  Building wheel for pyyaml (setup.py): started
  Building wheel for pyyaml (setup.py): finished with status 'error'
  ERROR: Complete output from command /usr/bin/python -u -c 'import setuptools, tokenize;__file__='"'"'/tmp/pip-install-5Bs3jo/pyyaml/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-e0qEDt --python-tag cp27:
  ERROR: running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.linux-x86_64-2.7
  creating build/lib.linux-x86_64-2.7/yaml
  copying lib/yaml/__init__.py -> build/lib.linux-x86_64-2.7/yaml
.
.
.
  copying lib/yaml/tokens.py -> build/lib.linux-x86_64-2.7/yaml
  running build_ext
  building '_yaml' extension
  creating build/temp.linux-x86_64-2.7
  creating build/temp.linux-x86_64-2.7/ext
  x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -fno-strict-aliasing -Wdate-time -D_FORTIFY_SOURCE=2 -g -fdebug-prefix-map=/build/python2.7-2.7.13=. -fstack-protector-strong -Wformat -Werror=format-security -fPIC -I/usr/include/python2.7 -c ext/_yaml.c -o build/temp.linux-x86_64-2.7/ext/_yaml.o
  In file included from ext/_yaml.c:596:0:
  ext/_yaml.h:2:18: fatal error: yaml.h: No such file or directory
   #include <yaml.h>
                    ^
  compilation terminated.
  Error compiling module, falling back to pure Python
  installing to build/bdist.linux-x86_64/wheel
  running install
  running install_lib
  creating build/bdist.linux-x86_64
  creating build/bdist.linux-x86_64/wheel
  creating build/bdist.linux-x86_64/wheel/yaml
.
.
.
  running install_egg_info
  running egg_info
  creating lib/PyYAML.egg-info
  writing lib/PyYAML.egg-info/PKG-INFO
  writing top-level names to lib/PyYAML.egg-info/top_level.txt
  writing dependency_links to lib/PyYAML.egg-info/dependency_links.txt
  writing manifest file 'lib/PyYAML.egg-info/SOURCES.txt'
  reading manifest file 'lib/PyYAML.egg-info/SOURCES.txt'
  writing manifest file 'lib/PyYAML.egg-info/SOURCES.txt'
  Copying lib/PyYAML.egg-info to build/bdist.linux-x86_64/wheel/PyYAML-5.3.1-py2.7.egg-info
  running install_scripts
  creating build/bdist.linux-x86_64/wheel/PyYAML-5.3.1.dist-info/WHEEL
  creating '/tmp/pip-wheel-e0qEDt/PyYAML-5.3.1-cp27-cp27mu-linux_x86_64.whl' and adding 'build/bdist.linux-x86_64/wheel' to it
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "/tmp/pip-install-5Bs3jo/pyyaml/setup.py", line 314, in <module>
      python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*',
    File "/usr/lib/python2.7/distutils/core.py", line 151, in setup
      dist.run_commands()
    File "/usr/lib/python2.7/distutils/dist.py", line 953, in run_commands
      self.run_command(cmd)
    File "/usr/lib/python2.7/distutils/dist.py", line 972, in run_command
      cmd_obj.run()
    File "/usr/local/lib/python2.7/dist-packages/wheel/bdist_wheel.py", line 254, in run
      wf.write_files(archive_root)
    File "/usr/local/lib/python2.7/dist-packages/wheel/wheelfile.py", line 124, in write_files
      self.write(path, arcname)
    File "/usr/local/lib/python2.7/dist-packages/wheel/wheelfile.py", line 135, in write
      zinfo = ZipInfo(arcname or filename, date_time=get_zipinfo_datetime(st.st_mtime))
    File "/usr/local/lib/python2.7/dist-packages/wheel/wheelfile.py", line 27, in get_zipinfo_datetime
      timestamp = int(os.environ.get('SOURCE_DATE_EPOCH', timestamp or time.time()))
  ValueError: invalid literal for int() with base 10: ''
```

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- HCO version (use `oc get csv -n kubevirt-hyperconverged`):
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- Install tools:
- Others:

--- a/hack/build-manifests.sh
+++ b/hack/build-manifests.sh
@@ -425,7 +425,7 @@ cp -f ${TEMPDIR}/*.${CRD_EXT} ${CRD_DIR}
 cp -f ${TEMPDIR}/*.${CRD_EXT} ${CSV_DIR}
 
 # Validate the yaml files
-pip install --user yamllint
+pip3 install --user yamllint
 (cd ${CRD_DIR} && yamllint -d "{extends: relaxed, rules: {line-length: disable}}" *.yaml)
 (cd ${CSV_DIR} && yamllint -d "{extends: relaxed, rules: {line-length: disable}}" *.yaml)
 

--- a/hack/build-manifests.sh
+++ b/hack/build-manifests.sh
@@ -425,9 +425,8 @@ cp -f ${TEMPDIR}/*.${CRD_EXT} ${CRD_DIR}
 cp -f ${TEMPDIR}/*.${CRD_EXT} ${CSV_DIR}
 
 # Validate the yaml files
-pip3 install --user yamllint
-(cd ${CRD_DIR} && yamllint -d "{extends: relaxed, rules: {line-length: disable}}" *.yaml)
-(cd ${CSV_DIR} && yamllint -d "{extends: relaxed, rules: {line-length: disable}}" *.yaml)
+(cd ${CRD_DIR} && docker run --rm -v "$(pwd)":/yaml quay.io/pusher/yamllint yamllint -d "{extends: relaxed, rules: {line-length: disable}}" /yaml)
+(cd ${CSV_DIR} && docker run --rm -v "$(pwd)":/yaml quay.io/pusher/yamllint yamllint -d "{extends: relaxed, rules: {line-length: disable}}" /yaml)
 
 # Check there are not API Groups overlap between different CNV operators
 ${PROJECT_ROOT}/tools/csv-merger/csv-merger --crds-dir=${CRD_DIR}

